internal/tokens/fftokens/fftokens.go

```diff
@@ -346,12 +354,15 @@ func (ft *FFTokens) handleTokenPoolCreate(ctx context.Context, data fftypes.JSON
 	blockchainEvent := data.GetObject("blockchain")
 
 	poolDataString := data.GetString("data")
-	if poolData == nil && poolDataString != "" {
```

Comment:
In unit testing, I found the `poolData == nil && poolDataString == ""` case was uncaught, and resulted in a panic

Reply:
https://github.com/hyperledger/firefly/issues/1121? 🙏